content/cli/kes-key/decrypt.md

```diff
@@ -8,42 +8,34 @@ tableOfContents: true
 
 ## Overview
 
-Decrypt the contents of a message with a key.
+Decrypt the contents of a data encryption key with a key and return the plaintext key.
```

Comment:
Does the text below sufficiently explain such that we could drop `with a key` here? There's a lot of keys in this one sentence.

Reply:
Reworked the sentence to add a few more instances of `key`.

I kid, I kid.
I reworked it to drop the `with a key` and replace the last one with `value`.